relayer/middleware/providers.middleware.ts

```diff
@@ -73,6 +86,9 @@ const defaultSupportedChains = {
     [CHAIN_ID_MOONBEAM]: {
       endpoints: ["https://rpc.testnet.moonbeam.network"],
     },
+    [CHAIN_ID_APTOS]: {
+      endpoints: ["https://fullnode.devnet.aptoslabs.com/v1"],
+    },
   },
   [Environment.DEVNET]: {},
```

Comment:
While we're here, what do you think about adding these to devnet since these are the two evm chains configured by default in tilt 
```
        [CHAIN_ID_ETH]: {
          endpoints: ["http://localhost:8545/"],
        },
        [CHAIN_ID_BSC]: {
          endpoints: ["http://localhost:8546/"],
        },
```